fhservices.cfg.j2

```diff
@@ -15,6 +15,11 @@ define command {
        command_line   $USER1$/check_http -H $HOSTADDRESS$ -S
 }
 
+define command {
+  command_name   check_mongodb
+  command_line   /opt/rhmap/nagios/plugins/check_mongodb_wrapper.sh -A $ARG1$ -S $ARG2$ -w $ARG3$ -c $ARG4$
```

Comment:
Why are the arguments passed as `$ARG1$`, `$ARG2$`, etc? Is it jinja2 syntax? Would it be possible to use named arguments instead of positional?